src/Native/System.Native/pal_runtimeinformation.h

```diff
@@ -14,5 +14,6 @@ enum
 {
     ARCH_X86,
     ARCH_X64,
-    ARCH_ARM
+    ARCH_ARM,
+    ARCH_ARM64
```

Comment:
Adding ARCH_ARM64 is fine, since it's not affecting the public RuntimeInformation type surface area. This System.Native native assembly is pinvoked by the ```System.Runtime.InteropServices.RuntimeInformation``` to get info for ```RuntimeInformation.OSArchitecture``` api. If you look [there](https://github.com/dotnet/corefx/blob/master/src/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/RuntimeInformation.Unix.cs#L31), we want to add another case to handle exception thrown for Arm64 value, like [this](https://github.com/Priya91/corefx-1/commit/5b1928421b8d427e0b3be59de63d68850054f0fc)